### DISCUSSION TITLE
Running AppImage from /opt

### DISCUSSION THREAD

**Discussion by delinuxco** [AppImage/AppImageKit#1342] [Q&A] [answered]
Just wondering if there are any issues/reasons that an AppImage could not be run from `/opt` ?

**Comment by probonopd** [comment, marked as answer]
You can run AppImages from (almost) any location, so `/opt` is fine. But you need root rights to put your AppImages there.

**Reply by delinuxco** [reply]
Excellent, I just wanted to make sure. I have been working on a little project with AppImage and I think I might be moving forward with this to help integrate AppImages into Linux system seemlessly to the user.

**Reply by probonopd** [reply]
For system integration, check https://github.com/AppImageCommunity/awesome-appimage?tab=readme-ov-file#desktop-integration.

**Reply by TheAssassin** [reply]
As a side note, AppImageLauncher supports a global `/Applications` directory.
